databricks/koalas/generic.py

```diff
@@ -1281,6 +1283,9 @@ def groupby(self, by, as_index: bool = True):
             raise ValueError("Grouper for '{}' not 1-dimensional".format(type(by)))
         if not len(by):
             raise ValueError('No group keys passed!')
+        axis = validate_axis(axis)
```

Comment:
if we're not going to implement `axis=1` for now, i think we don't need this yet.